casepro/msgs/views.py

```diff
@@ -186,7 +187,23 @@ def get_context_data(self, **kwargs):
             user = self.request.user
             page = int(self.request.GET.get('page', 1))
 
+            context['user_id'] = user.id
+
             search = self.derive_search()
+
+            # this is a refresh of messages
+            if self.request.GET.get('last_refresh', None):
+                new_messages = Message.search(org, user, search)
+
+                search['last_refresh'] = self.request.GET['last_refresh']
+                del search['after']
```

Comment:
The after parameter searches for messages where create date is greater than after, if a message created before our date is updated it is filtered out. The before date is fine.

It does make sense to move it to the Message.search function, I will do a commit now.